src/main/java/com/joyent/manta/cosbench/MantaStorage.java

```diff
@@ -305,10 +384,8 @@ public InputStream getObject(final String container, final String object, final
     }
 
     @Override
-    protected void createMetadata(final String container,
-                                  final String object,
-                                  final Map<String, String> map,
-                                  final Config config) {
+    protected void createMetadata(final String container, final String object, final Map<String, String> map,
```

Comment:
Long argument lists should be broken out on multiple lines (the one-arg-per-line is easier to read IMO)